deploy/osd-project-request-template/00-clusterrole.project-config.yaml

```diff
@@ -1,14 +0,0 @@
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  name: dedicated-admins-project-config
-rules:
-- apiGroups:
-  - config.openshift.io
-  resources:
-  - projects
```

Comment:
RBAC to add this back into the dedicated-admins-cluster-aggregate clusterrole needs added to this PR from https://github.com/openshift/managed-cluster-config/pull/524/files#diff-edf58d83a388b00c79c96f001db7431611c8eb13f3fef71a63073d499515c932R273-R280